PendingReleaseNotes.md

```diff
@@ -17,6 +17,7 @@
 - Recursive chown for mounts can now be toggled with the [ROOK_ENABLE_FSGROUP](https://github.com/rook/rook/issues/2254) environment variable.
 - Added the dashboard `port` configuration setting.
 - Added the dashboard `ssl` configuration setting.
+- Containers in `mon` pods have been removed and/or changed names.
```

Comment:
Could you add more specifics here about how the paths have changed inside the containers? For example, from /var/lib/rook to /var/lib/ceph